src/Diagnostics/LoggerBase.php

```diff
@@ -8,40 +8,18 @@
  */
 class LoggerBase
 {
-    /**
-     * Indicating whether enable logging
-     * @var bool Defaults to False
-    */
-    public $EnableLogging = false;
-   
-    /**
-     * Logging location
-     * @var bool Defaults to CoreConstants::DEFAULT_LOGGINGLOCATION
-     */
-    public $LoggingLocation = CoreConstants::DEFAULT_LOGGINGLOCATION;
-    
-    /**
-     * Log Level
-     * @var int Defaults to Off.  See TraceLevel
-     */
-    public $LoggingTraceLevel = TraceLevel::Off;
-    
+
     /**
      * Logs messages depending on the ids trace level.
      *
-     * @param int $idsTraceLevel IDS Trace Level.
+     * @param TraceLevel $idsTraceLevel IDS Trace Level.
      * @param string $messageToWrite The message to write.
      *
      */
     public function Log($idsTraceLevel, $messageToWrite)
     {
-        $fileToWrite = $this->LoggingLocation . '/executionlog.txt';
-        if(
-          $this->LoggingTraceLevel >= $idsTraceLevel &&
-          $this->EnableLogging &&
-          file_exists($fileToWrite) &&
-          is_writable($fileToWrite)
-        )
+        $fileToWrite = CoreConstants::DEFAULT_LOGGINGLOCATION . '/executionlog.txt';
```

Comment:
@abisalehalliprasan Can we not remove the idsTraceLevel parameter since it is no longer used.(Unless you have plans to use it later)